services/crypto_create.proto

```diff
@@ -151,4 +151,18 @@ message CryptoCreateTransactionBody {
      * If true, the account declines receiving a staking reward. The default value is false.
      */
     bool decline_reward = 17;
+
+    /**
+     * The bytes to be used as the account's alias. The bytes will be 1 of 2 options. 
+     * It will be the serialization of a protobuf Key message for an ED25519/ECDSA primitive key type. 
+     * Or the if the account is ECDSA based it may also be the public address, calcluated as the last 20 bytes of the keccak-256 of the ECDSA primitive key.
```

Comment:
```suggestion
     * Or the if the account is ECDSA based it may also be the public address, calcluated as the last 20 bytes of the keccak-256 hash of the ECDSA primitive key.
```

Reply:
Fixed